src/script/clean-ci

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# clean-ci [--dry-run] [--since="relative date"]
+#
+# Cleanup the ceph-ci.git repository branch list by deleting any branch with a
+# HEAD commit date older than (by default) 3 months.
+
+set -e
+
+CEPH_CI=https://github.com/ceph/ceph-ci.git
+DRY_RUN=0
+SINCE="3 months ago"
+
+
+function iterate_ci {
+    three_months_ago=$(date -d "$SINCE" +%s)
+
+    git ls-remote --refs --branches "$CEPH_CI" | \
```

Comment:
git v2.43.0 apparently doesn't have --branches

Reply:
The older equivalent is `--heads`.  It's no longer documented but still accepted by newer versions of git.

Reply:
fixed!